plat/common/include/uk/plat/common/irq.h

```diff
@@ -59,4 +59,6 @@ enum uk_irq_polarity {
 	UK_IRQ_POLARITY_MAX
 };
 
+void _ukplat_irq_handle(unsigned long irq);
+
```

Comment:
```suggestion
/**
 * Calls the registered IRQ handlers for the given IRQ vector. This will
 * acknowledge the IRQ.
 *
 * @param irq IRQ vector
 */
void _ukplat_irq_handle(unsigned long irq);
```